test/tests/adapters/mongoose.test.js

```diff
@@ -693,6 +693,36 @@ describe('adapters > schema-adapter', () => {
       expect(result).toHaveProperty('fields');
       expect(result.fields[0].isRequired).toStrictEqual(true);
     });
+
```

Comment:
I think the "isRequired flag" tests deserve some organisation improvements:

```javascript
describe('"isRequired" flag', () => {
  // Existing tests
  describe('on a field defined with "required" attribute set as "true"', () => {
    it('should define "isRequired" as "true"', () => {
      // ...
    });
  });

  describe('on a field defined without "required" attribute', () => {
    it('should not define "isRequired"', () => {
      // ...
    });
  });

  // Added tests of yours
  describe('on a non-generated "_id" field', () => {
    it('should not define "isRequired" as true', () => {
      // ...
    });
  });

  describe('on a generated "_id" field', () => {
    it('should not define "isRequired"', () => {
      // ...
    });
  });
});  
```

Reply:
Done, thanks